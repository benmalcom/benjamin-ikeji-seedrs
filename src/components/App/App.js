import React from 'react';
import { BrowserRouter, Route, Switch } from 'react-router-dom';

import Forecast from 'domain/WeatherInfo';

const App = () => (<div>
    <BrowserRouter>
      <Switch>
        <Route path="/" component={Forecast} />
      </Switch>
    </BrowserRouter>
  </div>
);

export default App;
