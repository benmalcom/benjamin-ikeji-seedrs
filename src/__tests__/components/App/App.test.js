import React from 'react';
import { render } from '@testing-library/react';
import App from 'components/App/App';
import { createTestStore } from 'utils/test/test-utils';
import { Provider } from 'react-redux';

describe('App', () => {
  it('renders properly', async () => {
    const component = render(<Provider store={createTestStore()}><App /></Provider>);
    expect(component).toMatchSnapshot();
  });

});