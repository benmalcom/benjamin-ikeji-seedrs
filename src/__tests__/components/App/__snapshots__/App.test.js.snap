// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg"
        >
          <div
            class="container"
          >
            <div
              aria-expanded="false"
              class="MuiAutocomplete-root MuiAutocomplete-hasClearIcon"
              role="combobox"
              style="width: 400px;"
            >
              <div
                class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
                  data-shrink="false"
                  for="google-map"
                  id="google-map-label"
                >
                  Search city...
                </label>
                <div
                  class="MuiInputBase-root MuiInput-root MuiInput-underline MuiAutocomplete-inputRoot MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
                >
                  <input
                    aria-autocomplete="both"
                    aria-invalid="false"
                    autocapitalize="none"
                    autocomplete="off"
                    class="MuiInputBase-input MuiInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused MuiInputBase-inputAdornedEnd"
                    id="google-map"
                    spellcheck="false"
                    type="text"
                    value=""
                  />
                  <div
                    class="MuiAutocomplete-endAdornment"
                  >
                    <button
                      aria-label="Clear"
                      class="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-clearIndicator"
                      tabindex="-1"
                      title="Clear"
                      type="button"
                    >
                      <span
                        class="MuiIconButton-label"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                          />
                        </svg>
                      </span>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiBox-root MuiBox-root-2 box"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-colorTextPrimary MuiTypography-gutterBottom MuiTypography-alignCenter"
            >
              No forecast items yet, use the search option above
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg"
      >
        <div
          class="container"
        >
          <div
            aria-expanded="false"
            class="MuiAutocomplete-root MuiAutocomplete-hasClearIcon"
            role="combobox"
            style="width: 400px;"
          >
            <div
              class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
                data-shrink="false"
                for="google-map"
                id="google-map-label"
              >
                Search city...
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiAutocomplete-inputRoot MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
              >
                <input
                  aria-autocomplete="both"
                  aria-invalid="false"
                  autocapitalize="none"
                  autocomplete="off"
                  class="MuiInputBase-input MuiInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused MuiInputBase-inputAdornedEnd"
                  id="google-map"
                  spellcheck="false"
                  type="text"
                  value=""
                />
                <div
                  class="MuiAutocomplete-endAdornment"
                >
                  <button
                    aria-label="Clear"
                    class="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-clearIndicator"
                    tabindex="-1"
                    title="Clear"
                    type="button"
                  >
                    <span
                      class="MuiIconButton-label"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-2 box"
        >
          <h6
            class="MuiTypography-root MuiTypography-h6 MuiTypography-colorTextPrimary MuiTypography-gutterBottom MuiTypography-alignCenter"
          >
            No forecast items yet, use the search option above
          </h6>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
